pkgs/development/compilers/llvm/13/clang/add-nostdlibinc-flag.patch

```diff
@@ -1,18 +0,0 @@
-diff --git a/lib/Driver/Driver.cpp b/lib/Driver/Driver.cpp
```

Comment:
nit, feel free to ignore: I think it'd be useful to have a comment on this patch file giving a little context about why this is required; maybe an abbreviated version of what's in [this comment](https://github.com/NixOS/nixpkgs/issues/191152#issuecomment-1429029003), something like:

```text
`--gcc-toolchain` causes `libstdc++` paths from `gcc` to be included, breaking `libc++`
`include_next` directives and we can't remove `--gcc-toolchain` when using `libc++`
because we may want to use `libgcc_s` with `libc++`; `--nostdlibinc` lets us retain the
`--gcc-toolchain` flag while keeping it from influencing include paths
```

Granted, I don't think we'll run into a situation like #194634 again for this patch but even so.

Reply:
I think this is adequately covered by my commit message, which I made quite thorough for exactly this reason.  IMO it's more useful to have a commit that you can checkout and be guaranteed to reproduce to understand the failure, than a comment that might go out of date.

Reply:
👍 That's fair; it took me a little while to realize why the flag was actually needed but agreed that the test case is more valuable; once you've got it, it's pretty straightforward to figure the rest out.